### PR TITLE
Update and release Cloud Tasks v2beta2

### PR DESCRIPTION
**Pull request by jskeet**

### CODE REVIEW

File: apis/Google.Cloud.Tasks.V2Beta2/Google.Cloud.Tasks.V2Beta2.Tests/CloudTasksClientTest.g.cs [hunk 1]
```diff
@@ -627,7 +631,7 @@ public void GetIamPolicy()
             Mock<CloudTasks.CloudTasksClient> mockGrpcClient = new Mock<CloudTasks.CloudTasksClient>(MockBehavior.Strict);
             GetIamPolicyRequest expectedRequest = new GetIamPolicyRequest
             {
-                Resource = new QueueName("[PROJECT]", "[LOCATION]", "[QUEUE]").ToString(),
+                ResourceAsResourceName = new ProjectName("[PROJECT]"),
```
Comment (jskeet) [inline apis/Google.Cloud.Tasks.V2Beta2/Google.Cloud.Tasks.V2Beta2.Tests/CloudTasksClientTest.g.cs]: We're expecting just a project name resource...

File: apis/Google.Cloud.Tasks.V2Beta2/Google.Cloud.Tasks.V2Beta2.Tests/CloudTasksClientTest.g.cs [hunk 2]
```diff
@@ -637,8 +641,8 @@ public void GetIamPolicy()
             mockGrpcClient.Setup(x => x.GetIamPolicyAsync(expectedRequest, It.IsAny<CallOptions>()))
                 .Returns(new Grpc.Core.AsyncUnaryCall<Policy>(Task.FromResult(expectedResponse), null, null, null, null));
             CloudTasksClient client = new CloudTasksClientImpl(mockGrpcClient.Object, null);
-            string formattedResource = new QueueName("[PROJECT]", "[LOCATION]", "[QUEUE]").ToString();
-            Policy response = await client.GetIamPolicyAsync(formattedResource);
+            QueueName resource = new QueueName("[PROJECT]", "[LOCATION]", "[QUEUE]");
```
Comment (jskeet) [inline apis/Google.Cloud.Tasks.V2Beta2/Google.Cloud.Tasks.V2Beta2.Tests/CloudTasksClientTest.g.cs]: ... but we're creating a queue name resource to pass in.

Comment (chrisdunelm) [reply apis/Google.Cloud.Tasks.V2Beta2/Google.Cloud.Tasks.V2Beta2.Tests/CloudTasksClientTest.g.cs]: Oh yes, not sure why that's happening. Will fix in a new PR.

**Comment by jskeet** [comment]
Closing this as we'll need to regenerate when codegen is fixed.

**Pull request closed without merging**